pynear/tests/integration/test_cli.py

```diff
@@ -0,0 +1,197 @@
+import json
+import os
+import random
+
+import delegator
+import pytest
```

Comment:
Just curious, is it an intentional choice of pytest over unittest?

Reply:
yep.. i did a bit of research. seems like pytest is a common choice by many major projects, seems like they are more feature rich, support functions as tests, have better reporting, etc.